### PR TITLE
Giving portal improvements

### PR DESCRIPTION
**Pull request by oetherington**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205845264040693) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
What's keeping this a draft?

File: packages/lesswrong/components/ea-forum/giving-portal/ElectionFundCTA.tsx [hunk 1]
```diff
@@ -4,6 +4,15 @@ import { ForumIconName } from "../../common/ForumIcon";
...
 const styles = (theme: ThemeType) => ({
+  /**
+   * This class is unused.
+   * Don't remove it or everything will break.
+   * It has to be the first class in this object.
+   * I don't have time to debug and this bug is making me nauseous.
+   */
+  unused: {
+    display: "flex",
+  },
```
Comment (jpaddison3) [inline packages/lesswrong/components/ea-forum/giving-portal/ElectionFundCTA.tsx:15]: This is the most cursed thing I've ever seen

Comment (jpaddison3) [reply packages/lesswrong/components/ea-forum/giving-portal/ElectionFundCTA.tsx:15]: I am losing it over here, @s-cheng can attest.

**Comment by oetherington** [comment]
@jpaddison3 Should be good to go now!

File: packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx [hunk 2]
```diff
@@ -164,14 +218,24 @@ const styles = (theme: ThemeType) => ({
...
+    [theme.breakpoints.down("sm")]: {
+      display: "none",
+    },
+  },
   mt10: { marginTop: 10 },
   mt20: { marginTop: 20 },
   mt30: { marginTop: 30 },
   mt60: { marginTop: 60 },
   mt80: { marginTop: 80 },
+  mb20: { marginBottom: 20 },
```
Comment (jpaddison3) [inline packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx:234]: Going full tailwind

File: packages/lesswrong/components/ea-forum/giving-portal/hooks.ts [hunk 3]
```diff
@@ -20,8 +20,9 @@ export const useDonationOpportunities = useElectionCandidates;
 export const useAmountRaised = () => {
   // TODO: Query for the actual amount
```
Comment (jpaddison3) [inline packages/lesswrong/components/ea-forum/giving-portal/hooks.ts:21]: Lol

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Pull request merged**